guide/english/python/itertools/index.md

```diff
@@ -58,4 +58,15 @@ combinations('ABCD', 2)
 combinations_with_replacement('ABCD', 2)	 	                                                     AA AB AC AD BB BC BD CC CD DD
 
 
-Source:https://docs.python.org/3/library/itertools.html
\ No newline at end of file
+Source:https://docs.python.org/3/library/itertools.html
+
+##Changes
+
+###compress()
+It make an iterator that filters elements from data returning only those that have a corresponding element in selectors that evaluates to True. Stops when either the data or selectors iterables has been exhausted.
+
+import itertools as i
+i.compress('ABCDEF', [1,0,1,0,1,1]) 
+--> A C E F
+
+Sourse : https://docs.python.org/2/library/itertools.html
```

Comment:
`Sourse` is incorrect. Please use the existing `Source:https://docs.python.org/3/library/itertools.html` that links to Python 3.